src/rhelocator/api/routes/health.py

```diff
@@ -0,0 +1,40 @@
+from flasgger import swag_from
+from flask import Blueprint
+from flask import Response
+
+
+def health_blueprint(data: dict[str, str]) -> Blueprint:
+    """Creates and returns a Flask blueprint that sanity checks the provided
+    image data.
+
+    Args:
+        data: Formatted image data dict.
+
+    Returns:
+        statuscode and message that describes operational health.
+    """
+    health_endpoint = Blueprint("health_api", __name__)
+
+    @health_endpoint.route("/health", methods=["GET"])
+    @swag_from("../schema/health.yml")
+    def endpoint() -> Response:
+        """Cloud Image Directory health Endpoint sanity checks the provided
+        image data."""
+
+        # TODO: Check-(check the value that comes back, ... ?)
+        message = "ok"
+        status = 200
+        image_data_is_corrupt = False
+
+        if data == None:
+            image_data_is_corrupt = True
+
+        if image_data_is_corrupt:
+            message = "image data is corrupt"
+            status = 500
+
+        return Response(
```

Comment:
For now I skipped the schema validation, because I saw that we currently run the json schema validation before creating the blueprints. What do you think?

Reply:
/api/health is fine for me.